### PR TITLE
fix(popover): fix the problem that popover display when the props'con…

### PR DESCRIPTION
**Pull request by DawnLck**
…tent is empty

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

NULL

### 💡 Background and solution

Popover 组件传入content为空时，还是会渲染出一个空的气泡，但是tooltip就不会，本次PR就是为了修复这个问题。
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Hide the empty popover when the props' content is undefined or null.        |
| 🇨🇳 Chinese |   修复当传入的内容参数和标题参数都为空时，气泡组件仍旧会渲染空气泡的问题    |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/popover/index.tsx [hunk 1]
```diff
@@ -13,12 +13,18 @@ const Popover = React.forwardRef<unknown, PopoverProps>(
...
 
-    const getOverlay = (prefixCls: string) => (
-      <>
-        {title && <div className={`${prefixCls}-title`}>{getRenderPropValue(title)}</div>}
-        <div className={`${prefixCls}-inner-content`}>{getRenderPropValue(content)}</div>
-      </>
-    );
+    const getOverlay = (prefixCls: string) => {
+      const isTitleEmpty = !title || title.toString() === '';
+      const isContentEmpty = !content || content.toString() === '';
```
Comment (MadCcc) [inline components/popover/index.tsx:18]: `title.toString() === ''`
是在判断什么？

Comment (DawnLck) [reply components/popover/index.tsx:18]: 判断是否为空字符串，如果是则不出现气泡

Comment (MadCcc) [reply components/popover/index.tsx:18]: 空字符串 `!title` 就可以判断出来吧，`toString` 如果用户传进来的东西没有那就会报错了

**Comment by MadCcc** [comment]
来个测试用例

File: components/popover/index.tsx (hunk 1 above)
Comment (DawnLck) [reply components/popover/index.tsx]: 了解，我更新一下PR

**Comment by DawnLck** [comment]
> 来个测试用例

```jsx
  <Popover content="">
    <Button type="primary">Hover me</Button>
  </Popover>
```

**Comment by MadCcc** [comment]
写到 test 里

**Comment by DawnLck** [comment]
> 写到 test 里

done

**Comment by MadCcc** [comment]
ci 看下，符合改动的 snapshot 可以提交上来

**Comment by DawnLck** [comment]
> ci 看下，符合改动的 snapshot 可以提交上来

已更新snapshot

**Comment by MadCcc** [comment]
Still some error with snapshot.

**Comment by DawnLck** [comment]
> Still some error with snapshot.

之前只更新了 popover 自身的snap，已在根目录下运行 jest --config .jest.js --cache=false -u 来更新所有受影响的snap

**Comment by DawnLck** [comment]
还不行的话，只能寻求帮助了

**Comment by DawnLck** [comment]
image

**Comment by MadCcc** [comment]
Some problem with dropdown, I'm working on it.

**Comment by MadCcc** [comment]
Please merge master.

**Comment by DawnLck** [comment]
> Please merge master.

done

**Comment by yoyo837** [comment]
请填写一下 Changelog，方便发版时工具抓取直接生成。

**Comment by DawnLck** [comment]
> 请填写一下 Changelog，方便发版时工具抓取直接生成。

done

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**